frontend/src/reducers/strapi.js

```diff
@@ -0,0 +1,83 @@
+import Sponsor1 from "../assets/IDPH logo.PNG";
+import Sponsor2 from "../assets/24thDistrict Police.PNG";
+import Sponsor3 from "../assets/Citron Hygiene.jpg";
+import Sponsor4 from "../assets/evanston Police.jpg";
+import Sponsor5 from "../assets/first united methodist Church.png";
+import Sponsor6 from "../assets/FlatTop Grill.png";
+import Sponsor7 from "../assets/HomersIceCreamLogo200.jpg";
+import Sponsor8 from "../assets/Kiwanis Logo (1).jpg";
+import Sponsor9 from "../assets/Logo2.jpg";
+import Sponsor10 from "../assets/Roger's Park Library.PNG";
+import Sponsor11 from "../assets/uic logo2.png";
+import Sponsor12 from "../assets/Walmart_logo.svg.png";
+
+const initialState = {
+  email: "founder@ymim.org",
+  phone: "773.941.1200",
+  taxId: "82-1904373",
+  donateLink: "https://www.paypal.com/",
+  token: "",
+  facebook: "https://www.facebook.com/theymim/",
+  instagram: "https://www.instagram.com/theyoungmasterbuilders/",
+  twitter: "https://twitter.com/YMIMtweets",
+  video1: "https://www.youtube.com/embed/PK9ESRMGq74",
+  video2: "https://www.youtube.com/embed/jdsqht1m1rE",
+  video3: "https://www.youtube.com/embed/GwXt3tL6FqY",
+  sponsor1: Sponsor1,
```

Comment:
this looks good in `GET_SPONSORS` action

but for initialState this is going to have to be `sponsors: [Sponsor1, Sponsor2, ...]` to work properly